lib/ch/row_binary.ex

```diff
@@ -363,8 +363,17 @@ defmodule Ch.RowBinary do
     def encode({unquote(enum_t), mapping}, e) do
       i =
         case e do
-          _ when is_integer(e) -> e
-          _ when is_binary(e) -> Map.fetch!(mapping, e)
+          _ when is_integer(e) ->
+            e
+
+          _ when is_binary(e) ->
+            case Map.fetch(mapping, e) do
+              {:ok, res} ->
+                res
+
+              :error ->
+                raise "The enum value #{e} was not present in the given enum mapping: #{inspect(mapping)}"
```

Comment:
Maybe wording and formatting can be closer to the `KeyError`

```elixir
iex(1)> Map.fetch! %{"hello" => 1, "world" => 2}, "hell"
** (KeyError) key "hell" not found in: %{"hello" => 1, "world" => 2}
    (stdlib 4.3) :maps.get("hell", %{"hello" => 1, "world" => 2})
    iex:1: (file)
```

Reply:
```elixir
raise ArgumentError, "enum value #{inspect(e)} not found in mapping: #{inspect(mapping)}"
```

Reply:
Sure will update! :)